pybind11/how-to.rst

```diff
@@ -166,82 +166,72 @@ Adding dependencies
 First we need to add some dependencies to the build.
 
 Scons will not use pybind11 unless it is setup, so in ``{{pkg}}/ups/{{pkg}}.table``,
-where ``{{pkg}}`` is the name of the package, you will need to add the dependency
-``setupRequired(pybind11)``.
+where ``{{pkg}}`` is the name of the package, you will need to add the dependency ``setupRequired(pybind11)``.
 You also need to modify the ``dependencies`` in ``{{pkg}}/ups/{{pkg}}.cfg``, by adding ``"pybind11"`` to ``"buildRequired"``.
 
 .. _creating-module-file:
 
 Creating a module file
 ^^^^^^^^^^^^^^^^^^^^^^
 
-Following :ref:`our rules on file naming <style-guide-pybind11-module-naming>`, we start by creating a minimal module file ``python/lsst/TMPL/exampleOne.cc`` with the following content:
+Following :ref:`our rules on file naming <style-guide-pybind11-module-naming>`, we start by creating a minimal module file ``python/lsst/TMPL/_tmpl.cc`` with the following content:
 
 .. code-block:: cpp
 
     #include "pybind11/pybind11.h"
 
+    #include "lsst/utils.python.h"
     #include "lsst/TMPL/ExampleOne.h"
 
     namespace py = pybind11;
 
     namespace lsst {
     namespace tmpl {
 
-    PYBIND11_MODULE(exampleOne, mod) {
-    }}}  // lsst::tmpl
+    PYBIND11_MODULE(_tmpl, mod) {
+        utils::python::WrapperCollection wrappers(mod, "lsst.tmpl");
+        // wrapper code goes here
+        wrappers.finish();
+    }
+
+    }}  // lsst::tmpl
+
+``WrapperCollection`` is a helper class provided by the LSST ``utils`` package that should be used in most LSST pybind11 wrappers.
```

Comment:
Is there ever a case when it shouldn't be used?

Reply:
I haven't thought of any, but I dislike being overly prescriptive in a how-to guide, especially having only wrapped one package with this so far.